### ISSUE TITLE
Hotspot for Menu Button

### ISSUE THREAD

**Issue by Incrested** [KnoxDevTeam/Cheter-IDE#4]
We should have the menu button disappear after a time and reappear when in the area of the menu button. I reckon we just need a div and something like the following to hide and appear (with an animation, of course).
```js
var isOnDiv = false;
$(yourDiv).mouseenter(function(){isOnDiv=true;});
$(yourDiv).mouseleave(function(){isOnDiv=false;});
```

Don't know if we should add jQuery or try to find a vanilla alternative.

**Comment by ialoam** [comment]
well you dont have to do that you can just put it in the top left of the text writing above the numbers

**Comment by Incrested** [comment]
Yeah, but at the moment it covers the preview box, which can make it a pain to try and see the content behind it and would just have people using something like Firefox when testing HTML files as usual: my aim is to mostly eliminate that: hence it's good to declutter the area when not needed, but still have it capable.

**Comment by ialoam** [comment]
![capture1](https://user-images.githubusercontent.com/40329123/43935079-29cc1148-9c07-11e8-9ba6-452bf712ca60.PNG)

**Comment by Incrested** [comment]
The issue with the line count aside, you can see it covers the words I had there.
![image](https://user-images.githubusercontent.com/30969346/43935181-a3d7bea6-9c07-11e8-8b7a-5bbfb35b64da.png)

**Comment by ialoam** [comment]
you can also change the opacity

**Comment by Incrested** [comment]
Yeah, I might just end up seeing to moving the entire balloon into the top bar menu when running in electron since we can have normal "File -> New" and etc. You reckon you can look into that while I look into the other issue of the line counter?

**Comment by Incrested** [comment]
Currently Fixing Up this Area to appear and disappear when in the area and adding an about page. Will close once committed (mid or late tomorrow at max)

**Comment by Incrested** [comment]
I'm thinking of some way to fix this in the future, such as adding a keystroke or smth to open the drawer and then leave the button to be static or to disappear and appear whenever you are near it; changing opacity would still keep it there, unless you change opacity when near to '100' and to '0' when away from it. Until then, I'll leave this open for suggestions and stuff.
